### PR TITLE
docs(readme): adapt for v2 release

### PR DESCRIPTION
**Pull request by kleberbaum**
- Get Up and Running in 5 Minutes
- Motivation
- How to Code
- Trivia
- Creators

Adapt the readme for the upcoming V2.0.0 release.

### CODE REVIEW

**Review by petute — APPROVED** [review]
Should be ready for merge.

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

Really awesome what you accomplished.
smooch <3

**Review by petschna — APPROVED** [review]
Should be ok to merge.

**Review by Aichnerc — APPROVED** [review]
Does the Pull Request description follow any template?

Content is looking good and should therefore be able to merged.

**Review by evilpro — APPROVED** [review]

**Review by simonprast — APPROVED** [review]
Thank you for implementing the improvements. Should be ready to merge now.

File: README.md [hunk 1]
```diff
@@ -486,6 +460,21 @@ to [contribute to our project](#contributing).
...
+#### Name:
+In Austria the first month of the year is called "Jänner" since we started working on this project in January we decided to name the project Jaen.
+#### Pronounciation:
+The name Jaen is pronounced (Jän)ner [ˈjɛn] or (Jan)uary [ˈdʒæn].
+#### Password:
+The standard password in Jaen is **ciscocisco**. The origin of this password were back at our time in school. Most of us went to school for network engineering and in the cisco courses the standard password would always be ciscocisco.
+#### Releases:
+Every one of our Jaen releases has it's own theme song. Have fun with it.
+#### Mascot:
+The inofficial mascot of this project is a girl holding an electric guitar.
```
Comment (schettn) [inline README.md:473]: Rlly? 👀

**Review by schettn — CHANGES_REQUESTED** [review]
have a look

**Pull request closed without merging**